iredis/entry.py

```diff
@@ -252,6 +252,7 @@ def repl(client, session, start_time):
     "-n", help="Database number.(overwrites dsn/url's db number)", default=None
 )
 @click.option("-a", "--password", help="Password to use when connecting to the server.")
+@click.option("-name", "--client_name", help="Name for current connection.")
```

Comment:
```suggestion
@click.option("-name", "--client_name", help="Assign a name to the current connection.")
```

From https://redis.io/commands/client-setname

Just to want to make this more clearly. :) hope you don't mind